### PR TITLE
Information about issue-priorities.md & new folder

### PR DESCRIPTION
**Pull request by adelinamart**
I created a new file to add the issue priorities and guidelines. 
I added a new folder for it so we can maybe move other files like contributing.md and developing.md under this one also. 

@rudygalfi for feedback also

### CODE REVIEW

File: Contributing/Issue_Priorities.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## AMP GitHub Issue Priorities
+The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need.
```
Comment (mrjoro) [inline Contributing/Issue_Priorities.md]: Since this documentation is intended for reference, I don't think it needs to refer to us "introducing" it or it being new.  We can send a separate announcement via the various channels once this is merged in.

File: Contributing/Issue_Priorities.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## AMP GitHub Issue Priorities
+The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
+These new guidelines can give you also a good overview of when to expect updates or closure of issues.
+
+In order to add a priority to your issues use our [list of GitHub labels](https://github.com/ampproject/amphtml/labels).
```
Comment (mrjoro) [inline Contributing/Issue_Priorities.md]: Are we expecting filers to set their own priority on issues?  If so, we should set the expectation that they are hints, but that there is a process for adjusting the priorities.  (We should also be very transparent for how those priorities are set.)

File: Contributing/Issue_Priorities.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### Priorities & Guidelines
+
+
+ Priority           | What it means          | Guidelines for Bugs              | Guidelines  for FRs               
+--------------------| ---------------------- | ---------------------------------| --------------------------------- 
+P0: Drop Everything | <ul><li>Outage</li><li>Critical production issue</li></ul>    | <ul><li>Drop everything until this is fixed</li><li>Should be assigned and accepted within 24 hours of filing</li><li>Provide regular bug update on status and ETA of the fix, once every day after acceptance</li><li>Fixed in 7 days (next release or patch release, whichever is sooner)</li></ul>  |  <ul><li>A FR is almost never a P0</li></ul>
```
Comment (mrjoro) [inline Contributing/Issue_Priorities.md]: "almost never" in "A FR is almost never a P0" feels like it leaves some wiggle room that I don't think is there.  I can't imagine a feature request that we'd tell people not to stop working on it until it was done, whereas a P0 may have that expectation.

File: Contributing/Issue_Priorities.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+P1: High Priority   | <ul><li>Breakage of a critical feature or user journey</li><li>Highly significant feature</li></ul>     | <ul><li>Should be assigned and accepted within 48 hours of filing</li><li>Should be updated by the team once every week after acceptance</li><li>Fixed within 30 days (next release) or updated with explanation and timeline</li></ul> | <ul><li>Implementation guidelines: 2 weeks</li><li>Feature requests are not P1, unless a situation develops where the lack of the feature is significant user problem</li></ul>
```
Comment (mrjoro) [inline Contributing/Issue_Priorities.md]: "Fixed within 30 days (next release)."  Are we saying "fixed in the next release, which will always be within 30 days?"  Or "fixed by the next release after 30 days."?  If the former I'd rephrase this as "fixed in the next release (within 30 days)."

For the FR part, "unless a situation develops where the lack of the feature is a significant user problem" is too ambiguous.  My sense is that it's quite common for someone filing a FR to feel like it's addressing a "significant user problem" so I think we need more stringent terminology here.

File: Contributing/Issue_Priorities.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+P2: Soon            | <ul><li>Breakage of a non-critical feature or user journey</li><li>Major usability problem (users frequently do the wrong thing)</li><li>Important feature</li></ul>   | <ul><li>Best effort</li><li>Fixed in a quarter</li></ul> | <ul><li>Implementation guidelines: 1 Quarter</li><li>Generally higher priority feature requests</li><li>These are mostly features that have been specifically scheduled to meet OKRs</li</ul>
```
Comment (mrjoro) [inline Contributing/Issue_Priorities.md]: I don't think we refer to OKRs anywhere else in the open source project, so we should remove this.

File: Contributing/Issue_Priorities.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+P1: High Priority   | <ul><li>Breakage of a critical feature or user journey</li><li>Highly significant feature</li></ul>     | <ul><li>Should be assigned and accepted within 48 hours of filing</li><li>Should be updated by the team once every week after acceptance</li><li>Fixed within 30 days (next release) or updated with explanation and timeline</li></ul> | <ul><li>Implementation guidelines: 2 weeks</li><li>Feature requests are not P1, unless a situation develops where the lack of the feature is significant user problem</li></ul>
+P2: Soon            | <ul><li>Breakage of a non-critical feature or user journey</li><li>Major usability problem (users frequently do the wrong thing)</li><li>Important feature</li></ul>   | <ul><li>Best effort</li><li>Fixed in a quarter</li></ul> | <ul><li>Implementation guidelines: 1 Quarter</li><li>Generally higher priority feature requests</li><li>These are mostly features that have been specifically scheduled to meet OKRs</li</ul> 
+P3: When Possible   | <ul><li>Minor usability problem (users are annoyed)</li><li>Polish</li><li>Minor features</li></ul> | <ul><li>Best effort</li></ul> | <ul><li>When possible</li></ul>
```
Comment (mrjoro) [inline Contributing/Issue_Priorities.md]: "Minor usability problem (users are annoyed)" might lead to some philosophical discussions ("is annoying users really a minor problem?")  I'd just leave this as "Minor usability problem."

File: Contributing/Issue_Priorities.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+<!---
```
Comment (cramforce) [inline Contributing/Issue_Priorities.md]: Please lowercase the file names are replace _ with -

File: Contributing/Issue_Priorities.md (hunk 1 above)
Comment (adelinamart) [inline Contributing/Issue_Priorities.md]: changed this

File: Contributing/Issue_Priorities.md (hunk 2 above)
Comment (adelinamart) [inline Contributing/Issue_Priorities.md]: Added a new sentence about this

File: Contributing/Issue_Priorities.md (hunk 3 above)
Comment (adelinamart) [inline Contributing/Issue_Priorities.md]: ok. Since didn't wanted to leave it blank added a new sentence

File: Contributing/Issue_Priorities.md (hunk 4 above)
Comment (adelinamart) [inline Contributing/Issue_Priorities.md]: ok. I added a general example, but I do want to leave room for some flexibility here on how to use this one (mainly at our end).

File: Contributing/Issue_Priorities.md (hunk 5 above)
Comment (adelinamart) [inline Contributing/Issue_Priorities.md]: ok. replaced with roadmap since they know about that.

File: Contributing/Issue_Priorities.md (hunk 6 above)
Comment (adelinamart) [inline Contributing/Issue_Priorities.md]: ok. removed

File: Contributing/Issue_Priorities.md (hunk 7 above)
Comment (adelinamart) [inline Contributing/Issue_Priorities.md]: done

File: contributing/issue-priorities.md [hunk 8]
```diff
@@ -15,18 +15,18 @@ limitations under the License.
 -->
 
 ## AMP GitHub Issue Priorities
-The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
+The AMP team is using the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
 These new guidelines can give you also a good overview of when to expect updates or closure of issues.
```
Comment (mrjoro) [inline contributing/issue-priorities.md]: "These guidelines" instead of "these new guidelines"

File: contributing/issue-priorities.md [hunk 9]
```diff
@@ -15,18 +15,18 @@ limitations under the License.
...
 
-In order to add a priority to your issues use our [list of GitHub labels](https://github.com/ampproject/amphtml/labels).
+In order to add a priority to your issues use one from the [list of GitHub labels](https://github.com/ampproject/amphtml/labels). Do expect the issue priority may change depending on the team's roadmap and backlog.
```
Comment (mrjoro) [inline contributing/issue-priorities.md]: Do we want to make it completely optional for the filer to set a priority?  In that case, I think we can phrase this as:

"If you would like to give a hint regarding the priority of the issue you can add the appropriate priority [GitHub label](https://github.com/ampproject/amphtml/labels).  Note that the priority may be changed as the issue is triaged based on the project's roadmap and backlog."

File: contributing/issue-priorities.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+--------------------| ---------------------- | ---------------------------------| --------------------------------- 
+P0: Drop Everything | <ul><li>Outage</li><li>Critical production issue</li></ul>    | <ul><li>Drop everything until this is fixed</li><li>Should be assigned and accepted within 24 hours of filing</li><li>Provide regular bug update on status and ETA of the fix, once every day after acceptance</li><li>Fixed in 7 days (next release or patch release, whichever is sooner)</li></ul>  |  <ul><li>We do not use P0 for FRs</li></ul>
+P1: High Priority   | <ul><li>Breakage of a critical feature or user journey</li><li>Highly significant feature</li></ul>     | <ul><li>Should be assigned and accepted within 48 hours of filing</li><li>Should be updated by the team once every week after acceptance</li><li>Fixed within 30 days or updated with explanation and timeline</li></ul> | <ul><li>Implementation guidelines: 2 weeks</li><li>Feature requests are not P1, unless a situation develops where the lack of the feature is significant user problem (eg. lack of the feature may block a significant number of AMP implementations)</li></ul>
```
Comment (mrjoro) [inline contributing/issue-priorities.md]: nit: change eg. to e.g.

**Comment by adelinamart** [comment]
Done with all the changes. Thanks for the feedback. Let me know if someone can merge this. Thanks.
